test/FunctionalTests/Tests/DataEdmLib/FunctionalUtilities/NavigationTestModelBuilder.cs

```diff
@@ -1793,9 +1771,9 @@ public static IEdmModel DerivedContainmentNavigationWithDerivedAndBaseAssociatio
             var officeSet = container.AddEntitySet("OfficeSet", office);
 
             officeSet.AddNavigationTarget(officeToEmployee, personSet);
-            personSet.AddNavigationTarget(officeToEmployee.Partner, officeSet);
+//            personSet.AddNavigationTarget(officeToEmployee.Partner, officeSet);

```

Comment:
remove, same for others

Reply:
I debated whether to remove or leave them. By spec it is invalid to set a nav prop binding on a containment navigation property, and I would have liked to have made this an error, but it would be a breaking change.  So I basically made it a no-op. Then I debated uncommenting the test code and just letting it run as-is and be a no-op, but I didn't really want to have code throughout the tests that looked like it was doing something, and might be a pattern others should follow, but was really a no-op.  So I left it in for now, but I'll go back and clean it up.